src/Nox.Types.Abstractions/Enums/NoxType.cs

```diff
@@ -85,7 +85,7 @@ public enum NoxType : uint
     [SimpleType(typeof(bool))]
     Boolean = 2157507194,
 
-    [SimpleType(typeof(string))]
+    [SimpleType(typeof(byte[]))]
```

Comment:
@gjorgji-grgovski-authoritypartners we will keep Color as string in dtos?

Reply:
so lets make this string again, so we can merge this PR

@gjorgji-grgovski-authoritypartners  is taking care of creating a tickets to refactor Color Value Object to be string and a converter to varchar(9)